src/settings.c

```diff
@@ -423,6 +423,21 @@ void load_settings(char *cmdline_config_path)
                 g_free(c);
         }
 
+        {
+                char *c = option_get_string(
+                        "global",
+                        "content_alignment", "-content_alignment", "center",
+                        "Align icon and text top/center/bottom"
+                );
+                if (!string_parse_content_alignment(c, &settings.content_alignment)) {
+                        if (c)
+                                LOG_W("Unknown content alignment: '%s'", c);
+                        settings.content_alignment = defaults.content_alignment;
```

Comment:
You forgot to initialize `defaults.content_alignment` (`config.h` in root).

Reply:
Ha. I looked for it, but didn't find it. Will do so.